scripts/install/all.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+MOZITP="/home/vagrant/MozITP"
```

Comment:
Can we have a global `config.sh` or something that contains this kind of global env variable. And in every script we can `source config.sh` to get them. 

You can see `MOZITP` appears in `scripts/install/system/provision.sh` again, and I believe we can use it in many other scripts as well.  The `config.sh` can contain things like default system path, default node/npm version, etc.